<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.269">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Bibliography</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


<link rel="stylesheet" href="biblio.css">
</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-full">

<main class="content column-page" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Bibliography</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>



<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-10x2018pbmc" class="csl-entry" role="doc-biblioentry">
10x Genomics. (2018). 1k PBMCs from a healthy donor (v3 chemistry). <a href="https://www.10xgenomics.com/resources/datasets/1-k-pbm-cs-from-a-healthy-donor-v-3-chemistry-3-standard-3-0-0">https://www.10xgenomics.com/resources/datasets/1-k-pbm-cs-from-a-healthy-donor-v-3-chemistry-3-standard-3-0-0</a>
</div>
<div id="ref-10x2019pbmc" class="csl-entry" role="doc-biblioentry">
10x Genomics. (2019). 5k peripheral blood mononuclear cells (PBMCs) from a healthy donor with a panel of TotalSeq-b antibodies (v3 chemistry). <a href="https://www.10xgenomics.com/resources/datasets/5-k-peripheral-blood-mononuclear-cells-pbm-cs-from-a-healthy-donor-with-cell-surface-proteins-v-3-chemistry-3-1-standard-3-1-0">https://www.10xgenomics.com/resources/datasets/5-k-peripheral-blood-mononuclear-cells-pbm-cs-from-a-healthy-donor-with-cell-surface-proteins-v-3-chemistry-3-1-standard-3-1-0</a>
</div>
<div id="ref-agrawal2021mde" class="csl-entry" role="doc-biblioentry">
Agrawal, A., Ali, A., &amp; Boyd, S. (2021). Minimum-distortion embedding. <em>Foundations and Trends<span></span> in Machine Learning</em>, <em>14</em>(3), 211–378. <a href="https://doi.org/10.1561/2200000090">https://doi.org/10.1561/2200000090</a>
</div>
<div id="ref-andersson2020single" class="csl-entry" role="doc-biblioentry">
Andersson, A., Bergenstråhle, J., Asp, M., Bergenstråhle, L., Jurek, A., Navarro, J. F., &amp; Lundeberg, J. (2020). Single-cell and spatial transcriptomics enables probabilistic inference of cell type topography. <em>Communications Biology</em>, <em>3</em>(1). <a href="https://doi.org/10.1038/s42003-020-01247-y">https://doi.org/10.1038/s42003-020-01247-y</a>
</div>
<div id="ref-batson2019molecular" class="csl-entry" role="doc-biblioentry">
Batson, J., Royer, L., &amp; Webber, J. (2019). Molecular cross-validation for single-cell RNA-seq. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/786269">https://doi.org/10.1101/786269</a>
</div>
<div id="ref-biancalani2021deep" class="csl-entry" role="doc-biblioentry">
Biancalani, T., Scalia, G., Buffoni, L., Avasthi, R., Lu, Z., Sanger, A., et al. (2021). Deep learning and alignment of spatially resolved single-cell transcriptomes with tangram. <em>Nature Methods</em>, <em>18</em>(11), 1352–1362. <a href="https://doi.org/10.1038/s41592-021-01264-7">https://doi.org/10.1038/s41592-021-01264-7</a>
</div>
<div id="ref-bland2000odds" class="csl-entry" role="doc-biblioentry">
Bland, J. M. (2000). Statistics notes: The odds ratio. <em><span>BMJ</span></em>, <em>320</em>(7247), 1468–1468. <a href="https://doi.org/10.1136/bmj.320.7247.1468">https://doi.org/10.1136/bmj.320.7247.1468</a>
</div>
<div id="ref-bttner2018test" class="csl-entry" role="doc-biblioentry">
Büttner, M., Miao, Z., Wolf, F. A., Teichmann, S. A., &amp; Theis, F. J. (2018). A test metric for assessing single-cell <span>RNA</span>-seq batch correction. <em>Nature Methods</em>, <em>16</em>(1), 43–49. <a href="https://doi.org/10.1038/s41592-018-0254-1">https://doi.org/10.1038/s41592-018-0254-1</a>
</div>
<div id="ref-cabello2020singlecellsignalr" class="csl-entry" role="doc-biblioentry">
Cabello-Aguilar, S., Alame, M., Kon-Sun-Tack, F., Fau, C., Lacroix, M., &amp; Colinge, J. (2020). <span>SingleCellSignalR</span>: Inference of intercellular networks from single-cell transcriptomics. <em>Nucleic Acids Research</em>, <em>48</em>(10), e55–e55. <a href="https://doi.org/10.1093/nar/gkaa183">https://doi.org/10.1093/nar/gkaa183</a>
</div>
<div id="ref-cable2021robust" class="csl-entry" role="doc-biblioentry">
Cable, D. M., Murray, E., Zou, L. S., Goeva, A., Macosko, E. Z., Chen, F., &amp; Irizarry, R. A. (2021). Robust decomposition of cell type mixtures in spatial transcriptomics. <em>Nature Biotechnology</em>, <em>40</em>(4), 517–526. <a href="https://doi.org/10.1038/s41587-021-00830-w">https://doi.org/10.1038/s41587-021-00830-w</a>
</div>
<div id="ref-cao2018joint" class="csl-entry" role="doc-biblioentry">
Cao, J., Cusanovich, D. A., Ramani, V., Aghamirzaie, D., Pliner, H. A., Hill, A. J., et al. (2018). Joint profiling of chromatin accessibility and gene expression in thousands of single cells. <em>Science</em>, <em>361</em>(6409), 1380–1385. <a href="https://doi.org/10.1126/science.aau0730">https://doi.org/10.1126/science.aau0730</a>
</div>
<div id="ref-cao2020human" class="csl-entry" role="doc-biblioentry">
Cao, J., O’Day, D. R., Pliner, H. A., Kingsley, P. D., Deng, M., Daza, R. M., et al. (2020). A human cell atlas of fetal gene expression. <em>Science</em>, <em>370</em>(6518). <a href="https://doi.org/10.1126/science.aba7721">https://doi.org/10.1126/science.aba7721</a>
</div>
<div id="ref-chen2009local" class="csl-entry" role="doc-biblioentry">
Chen, L., &amp; Buja, A. (2009). Local multidimensional scaling for nonlinear dimension reduction, graph drawing, and proximity analysis. <em>Journal of the American Statistical Association</em>, <em>104</em>(485), 209–219. <a href="https://doi.org/10.1198/jasa.2009.0111">https://doi.org/10.1198/jasa.2009.0111</a>
</div>
<div id="ref-chen2016xgboost" class="csl-entry" role="doc-biblioentry">
Chen, T., &amp; Guestrin, C. (2016). <span>XGBoost</span>. In <em>Proceedings of the 22nd <span>ACM</span> <span>SIGKDD</span> international conference on knowledge discovery and data mining</em>. <span>Acm</span>. <a href="https://doi.org/10.1145/2939672.2939785">https://doi.org/10.1145/2939672.2939785</a>
</div>
<div id="ref-cichocki2009fast" class="csl-entry" role="doc-biblioentry">
Cichocki, A., &amp; Phan, A.-H. (2009). Fast local algorithms for large scale nonnegative matrix and tensor factorizations. <em><span>IEICE</span> Transactions on Fundamentals of Electronics, Communications and Computer Sciences</em>, <em>E92-a</em>(3), 708–721. <a href="https://doi.org/10.1587/transfun.e92.a.708">https://doi.org/10.1587/transfun.e92.a.708</a>
</div>
<div id="ref-coifman2006diffusion" class="csl-entry" role="doc-biblioentry">
Coifman, R. R., &amp; Lafon, S. (2006). Diffusion maps. <em>Applied and Computational Harmonic Analysis</em>, <em>21</em>(1), 5–30. <a href="https://doi.org/10.1016/j.acha.2006.04.006">https://doi.org/10.1016/j.acha.2006.04.006</a>
</div>
<div id="ref-cover1967nearest" class="csl-entry" role="doc-biblioentry">
Cover, T., &amp; Hart, P. (1967). Nearest neighbor pattern classification. <em><span>IEEE</span> Transactions on Information Theory</em>, <em>13</em>(1), 21–27. <a href="https://doi.org/10.1109/tit.1967.1053964">https://doi.org/10.1109/tit.1967.1053964</a>
</div>
<div id="ref-davis2006prauc" class="csl-entry" role="doc-biblioentry">
Davis, J., &amp; Goadrich, M. (2006). The relationship between precision-recall and <span>ROC</span> curves. In <em>Proceedings of the 23rd international conference on machine learning - <span>ICML</span> <span>'</span>06</em>. <span>ACM</span> Press. <a href="https://doi.org/10.1145/1143844.1143874">https://doi.org/10.1145/1143844.1143874</a>
</div>
<div id="ref-van2018recovering" class="csl-entry" role="doc-biblioentry">
Dijk, D. van, Sharma, R., Nainys, J., Yim, K., Kathail, P., Carr, A. J., et al. (2018). Recovering gene interactions from single-cell data using data diffusion. <em>Cell</em>, <em>174</em>(3), 716–729.e27. <a href="https://doi.org/10.1016/j.cell.2018.05.061">https://doi.org/10.1016/j.cell.2018.05.061</a>
</div>
<div id="ref-dimitrov2022comparison" class="csl-entry" role="doc-biblioentry">
Dimitrov, D., Türei, D., Garrido-Rodriguez, M., Burmedi, P. L., Nagai, J. S., Boys, C., et al. (2022). Comparison of methods and resources for cell-cell communication inference from single-cell <span>RNA</span>-seq data. <em>Nature Communications</em>, <em>13</em>(1). <a href="https://doi.org/10.1038/s41467-022-30755-0">https://doi.org/10.1038/s41467-022-30755-0</a>
</div>
<div id="ref-efremova2020cellphonedb" class="csl-entry" role="doc-biblioentry">
Efremova, M., Vento-Tormo, M., Teichmann, S. A., &amp; Vento-Tormo, R. (2020). <span>CellPhoneDB</span>: Inferring cell–cell communication from combined expression of multi-subunit ligand–receptor complexes. <em>Nature Protocols</em>, <em>15</em>(4), 1484–1506. <a href="https://doi.org/10.1038/s41596-020-0292-x">https://doi.org/10.1038/s41596-020-0292-x</a>
</div>
<div id="ref-eraslan2019single" class="csl-entry" role="doc-biblioentry">
Eraslan, G., Simon, L. M., Mircea, M., Mueller, N. S., &amp; Theis, F. J. (2019). Single-cell <span>RNA</span>-seq denoising using a deep count autoencoder. <em>Nature Communications</em>, <em>10</em>(1). <a href="https://doi.org/10.1038/s41467-018-07931-2">https://doi.org/10.1038/s41467-018-07931-2</a>
</div>
<div id="ref-gower1975generalized" class="csl-entry" role="doc-biblioentry">
Gower, J. C. (1975). Generalized procrustes analysis. <em>Psychometrika</em>, <em>40</em>(1), 33–51. <a href="https://doi.org/10.1007/bf02291478">https://doi.org/10.1007/bf02291478</a>
</div>
<div id="ref-grandini2020metrics" class="csl-entry" role="doc-biblioentry">
Grandini, M., Bagli, E., &amp; Visani, G. (2020). Metrics for multi-class classification: An overview. <em>arXiv</em>. <a href="https://doi.org/10.48550/arxiv.2008.05756">https://doi.org/10.48550/arxiv.2008.05756</a>
</div>
<div id="ref-granja2021archr" class="csl-entry" role="doc-biblioentry">
Granja, J. M., Corces, M. R., Pierce, S. E., Bagdatli, S. T., Choudhry, H., Chang, H. Y., &amp; Greenleaf, W. J. (2021). <span>ArchR</span> is a scalable software package for integrative single-cell chromatin accessibility analysis. <em>Nature Genetics</em>, <em>53</em>(3), 403–411. <a href="https://doi.org/10.1038/s41588-021-00790-6">https://doi.org/10.1038/s41588-021-00790-6</a>
</div>
<div id="ref-grn2014validation" class="csl-entry" role="doc-biblioentry">
Grün, D., Kester, L., &amp; Oudenaarden, A. van. (2014). Validation of noise models for single-cell transcriptomics. <em>Nature Methods</em>, <em>11</em>(6), 637–640. <a href="https://doi.org/10.1038/nmeth.2930">https://doi.org/10.1038/nmeth.2930</a>
</div>
<div id="ref-haghverdi2018batch" class="csl-entry" role="doc-biblioentry">
Haghverdi, L., Lun, A. T. L., Morgan, M. D., &amp; Marioni, J. C. (2018). Batch effects in single-cell <span>RNA</span>-sequencing data are corrected by matching mutual nearest neighbors. <em>Nature Biotechnology</em>, <em>36</em>(5), 421–427. <a href="https://doi.org/10.1038/nbt.4091">https://doi.org/10.1038/nbt.4091</a>
</div>
<div id="ref-hammarlund2018cengen" class="csl-entry" role="doc-biblioentry">
Hammarlund, M., Hobert, O., Miller, D. M., &amp; Sestan, N. (2018). The <span>CeNGEN</span> project: The complete gene expression map of an entire nervous system. <em>Neuron</em>, <em>99</em>(3), 430–433. <a href="https://doi.org/10.1016/j.neuron.2018.07.042">https://doi.org/10.1016/j.neuron.2018.07.042</a>
</div>
<div id="ref-hao2021integrated" class="csl-entry" role="doc-biblioentry">
Hao, Y., Hao, S., Andersen-Nissen, E., Mauck, W. M., Zheng, S., Butler, A., et al. (2021). Integrated analysis of multimodal single-cell data. <em>Cell</em>, <em>184</em>(13), 3573–3587.e29. <a href="https://doi.org/10.1016/j.cell.2021.04.048">https://doi.org/10.1016/j.cell.2021.04.048</a>
</div>
<div id="ref-hie2019efficient" class="csl-entry" role="doc-biblioentry">
Hie, B., Bryson, B., &amp; Berger, B. (2019). Efficient integration of heterogeneous single-cell transcriptomes using scanorama. <em>Nature Biotechnology</em>, <em>37</em>(6), 685–691. <a href="https://doi.org/10.1038/s41587-019-0113-3">https://doi.org/10.1038/s41587-019-0113-3</a>
</div>
<div id="ref-hinton1989connectionist" class="csl-entry" role="doc-biblioentry">
Hinton, G. E. (1989). Connectionist learning procedures. <em>Artificial Intelligence</em>, <em>40</em>(1-3), 185–234. <a href="https://doi.org/10.1016/0004-3702(89)90049-0">https://doi.org/10.1016/0004-3702(89)90049-0</a>
</div>
<div id="ref-hosmer2013applied" class="csl-entry" role="doc-biblioentry">
Hosmer Jr, D. W., Lemeshow, S., &amp; Sturdivant, R. X. (2013). <em>Applied logistic regression</em> (Vol. 398). John Wiley &amp; Sons.
</div>
<div id="ref-hou2019scmatch" class="csl-entry" role="doc-biblioentry">
Hou, R., Denisenko, E., &amp; Forrest, A. R. R. (2019). <span class="nocase">scMatch</span>: A single-cell gene expression profile annotation tool using reference datasets. <em>Bioinformatics</em>, <em>35</em>(22), 4688–4695. <a href="https://doi.org/10.1093/bioinformatics/btz292">https://doi.org/10.1093/bioinformatics/btz292</a>
</div>
<div id="ref-hou2020predicting" class="csl-entry" role="doc-biblioentry">
Hou, R., Denisenko, E., Ong, H. T., Ramilowski, J. A., &amp; Forrest, A. R. R. (2020). Predicting cell-to-cell communication networks using <span>NATMI</span>. <em>Nature Communications</em>, <em>11</em>(1). <a href="https://doi.org/10.1038/s41467-020-18873-z">https://doi.org/10.1038/s41467-020-18873-z</a>
</div>
<div id="ref-hou2020systematic" class="csl-entry" role="doc-biblioentry">
Hou, W., Ji, Z., Ji, H., &amp; Hicks, S. C. (2020). A systematic evaluation of single-cell <span>RNA</span>-sequencing imputation methods. <em>Genome Biology</em>, <em>21</em>(1). <a href="https://doi.org/10.1186/s13059-020-02132-x">https://doi.org/10.1186/s13059-020-02132-x</a>
</div>
<div id="ref-hansen2012removing" class="csl-entry" role="doc-biblioentry">
Johnson, W. E., Li, C., &amp; Rabinovic, A. (2006). Adjusting batch effects in microarray expression data using empirical bayes methods. <em>Biostatistics</em>, <em>8</em>(1), 118–127. <a href="https://doi.org/10.1093/biostatistics/kxj037">https://doi.org/10.1093/biostatistics/kxj037</a>
</div>
<div id="ref-kiselev2019challenges" class="csl-entry" role="doc-biblioentry">
Kiselev, V. Y., Andrews, T. S., &amp; Hemberg, M. (2019). Challenges in unsupervised clustering of single-cell <span>RNA</span>-seq data. <em>Nature Reviews Genetics</em>, <em>20</em>(5), 273–282. <a href="https://doi.org/10.1038/s41576-018-0088-9">https://doi.org/10.1038/s41576-018-0088-9</a>
</div>
<div id="ref-kleshchevnikov2022cell2location" class="csl-entry" role="doc-biblioentry">
Kleshchevnikov, V., Shmatko, A., Dann, E., Aivazidis, A., King, H. W., Li, T., et al. (2022). Cell2location maps fine-grained cell types in spatial transcriptomics. <em>Nature Biotechnology</em>, <em>40</em>(5), 661–671. <a href="https://doi.org/10.1038/s41587-021-01139-4">https://doi.org/10.1038/s41587-021-01139-4</a>
</div>
<div id="ref-korsunsky2019fast" class="csl-entry" role="doc-biblioentry">
Korsunsky, I., Millard, N., Fan, J., Slowikowski, K., Zhang, F., Wei, K., et al. (2019). Fast, sensitive and accurate integration of single-cell data with harmony. <em>Nature Methods</em>, <em>16</em>(12), 1289–1296. <a href="https://doi.org/10.1038/s41592-019-0619-0">https://doi.org/10.1038/s41592-019-0619-0</a>
</div>
<div id="ref-kraemer2018dimred" class="csl-entry" role="doc-biblioentry">
Kraemer, G., Reichstein, M., &amp; Miguel, D. M. (2018). <span class="nocase">dimRed</span> and <span class="nocase">coRanking</span> - unifying dimensionality reduction in r. <em>The R Journal</em>, <em>10</em>(1), 342. <a href="https://doi.org/10.32614/rj-2018-039">https://doi.org/10.32614/rj-2018-039</a>
</div>
<div id="ref-kruskal1964mds" class="csl-entry" role="doc-biblioentry">
Kruskal, J. B. (1964). Multidimensional scaling by optimizing goodness of fit to a nonmetric hypothesis. <em>Psychometrika</em>, <em>29</em>(1), 1–27. <a href="https://doi.org/10.1007/bf02289565">https://doi.org/10.1007/bf02289565</a>
</div>
<div id="ref-lance2022multimodal" class="csl-entry" role="doc-biblioentry">
Lance, C., Luecken, M. D., Burkhardt, D. B., Cannoodt, R., Rautenstrauch, P., Laddach, A., et al. (2022). Multimodal single cell data integration challenge: Results and lessons learned. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2022.04.11.487796">https://doi.org/10.1101/2022.04.11.487796</a>
</div>
<div id="ref-lawson1995solving" class="csl-entry" role="doc-biblioentry">
Lawson, C. L., &amp; Hanson, R. J. (1995). <em>Solving least squares problems</em>. Society for Industrial; Applied Mathematics. <a href="https://doi.org/10.1137/1.9781611971217">https://doi.org/10.1137/1.9781611971217</a>
</div>
<div id="ref-lee2009quality" class="csl-entry" role="doc-biblioentry">
Lee, J. A., &amp; Verleysen, M. (2009). Quality assessment of dimensionality reduction: Rank-based criteria. <em>Neurocomputing</em>, <em>72</em>(7-9), 1431–1443. <a href="https://doi.org/10.1016/j.neucom.2008.12.017">https://doi.org/10.1016/j.neucom.2008.12.017</a>
</div>
<div id="ref-linderman2018zero" class="csl-entry" role="doc-biblioentry">
Linderman, G. C., Zhao, J., &amp; Kluger, Y. (2018). Zero-preserving imputation of scRNA-seq data using low-rank approximation. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/397588">https://doi.org/10.1101/397588</a>
</div>
<div id="ref-lopez2022destvi" class="csl-entry" role="doc-biblioentry">
Lopez, R., Li, B., Keren-Shaul, H., Boyeau, P., Kedmi, M., Pilzer, D., et al. (2022). <span>DestVI</span> identifies continuums of cell types in spatial transcriptomics data. <em>Nature Biotechnology</em>, <em>40</em>(9), 1360–1369. <a href="https://doi.org/10.1038/s41587-022-01272-8">https://doi.org/10.1038/s41587-022-01272-8</a>
</div>
<div id="ref-lopez2018deep" class="csl-entry" role="doc-biblioentry">
Lopez, R., Regier, J., Cole, M. B., Jordan, M. I., &amp; Yosef, N. (2018). Deep generative modeling for single-cell transcriptomics. <em>Nature Methods</em>, <em>15</em>(12), 1053–1058. <a href="https://doi.org/10.1038/s41592-018-0229-2">https://doi.org/10.1038/s41592-018-0229-2</a>
</div>
<div id="ref-lotfollahi2020query" class="csl-entry" role="doc-biblioentry">
Lotfollahi, M., Naghipourfar, M., Luecken, M. D., Khajavi, M., Büttner, M., Avsec, Z., et al. (2020). Query to reference single-cell integration with transfer learning. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/2020.07.16.205997">https://doi.org/10.1101/2020.07.16.205997</a>
</div>
<div id="ref-luecken2022benchmarking" class="csl-entry" role="doc-biblioentry">
Luecken, M. D., Büttner, M., Chaichoompu, K., Danese, A., Interlandi, M., Mueller, M. F., et al. (2021). Benchmarking atlas-level data integration in single-cell genomics. <em>Nature Methods</em>, <em>19</em>(1), 41–50. <a href="https://doi.org/10.1038/s41592-021-01336-8">https://doi.org/10.1038/s41592-021-01336-8</a>
</div>
<div id="ref-lueks2011evaluate" class="csl-entry" role="doc-biblioentry">
Lueks, W., Mokbel, B., Biehl, M., &amp; Hammer, B. (2011). How to evaluate dimensionality reduction? - improving the co-ranking matrix. <em>arXiv</em>. <a href="https://doi.org/10.48550/ARXIV.1110.3917">https://doi.org/10.48550/ARXIV.1110.3917</a>
</div>
<div id="ref-lun2019fastmnn" class="csl-entry" role="doc-biblioentry">
Lun, A. (2019). A description of the theory behind the fastMNN algorithm. <a href="https://marionilab.github.io/FurtherMNN2018/theory/description.html">https://marionilab.github.io/FurtherMNN2018/theory/description.html</a>
</div>
<div id="ref-mcinnes2018umap" class="csl-entry" role="doc-biblioentry">
McInnes, L., Healy, J., &amp; Melville, J. (2018). UMAP: Uniform manifold approximation and projection for dimension reduction. <em>arXiv</em>. <a href="https://doi.org/10.48550/arxiv.1802.03426">https://doi.org/10.48550/arxiv.1802.03426</a>
</div>
<div id="ref-miles2005rsquared" class="csl-entry" role="doc-biblioentry">
Miles, J. (2005). Encyclopedia of statistics in behavioral science. In. John Wiley <span>&amp;</span> Sons, Ltd. <a href="https://doi.org/10.1002/0470013192.bsa526">https://doi.org/10.1002/0470013192.bsa526</a>
</div>
<div id="ref-moon2019visualizing" class="csl-entry" role="doc-biblioentry">
Moon, K. R., Dijk, D. van, Wang, Z., Gigante, S., Burkhardt, D. B., Chen, W. S., et al. (2019). Visualizing structure and transitions in high-dimensional biological data. <em>Nature Biotechnology</em>, <em>37</em>(12), 1482–1492. <a href="https://doi.org/10.1038/s41587-019-0336-3">https://doi.org/10.1038/s41587-019-0336-3</a>
</div>
<div id="ref-narayan2021assessing" class="csl-entry" role="doc-biblioentry">
Narayan, A., Berger, B., &amp; Cho, H. (2021). Assessing single-cell transcriptomic variability through density-preserving data visualization. <em>Nature Biotechnology</em>, <em>39</em>(6), 765–774. <a href="https://doi.org/10.1038/s41587-020-00801-7">https://doi.org/10.1038/s41587-020-00801-7</a>
</div>
<div id="ref-nestorowa2016single" class="csl-entry" role="doc-biblioentry">
Nestorowa, S., Hamey, F. K., Sala, B. P., Diamanti, E., Shepherd, M., Laurenti, E., et al. (2016). A single-cell resolution map of mouse hematopoietic stem and progenitor cell differentiation. <em>Blood</em>, <em>128</em>(8), e20–e31. <a href="https://doi.org/10.1182/blood-2016-05-716480">https://doi.org/10.1182/blood-2016-05-716480</a>
</div>
<div id="ref-olsson2016single" class="csl-entry" role="doc-biblioentry">
Olsson, A., Venkatasubramanian, M., Chaudhri, V. K., Aronow, B. J., Salomonis, N., Singh, H., &amp; Grimes, H. L. (2016). Single-cell analysis of mixed-lineage states leading to a binary cell fate choice. <em>Nature</em>, <em>537</em>(7622), 698–702. <a href="https://doi.org/10.1038/nature19348">https://doi.org/10.1038/nature19348</a>
</div>
<div id="ref-openproblems" class="csl-entry" role="doc-biblioentry">
Open Problems for Single Cell Analysis Consortium. (2022). Open problems. <a href="https://openproblems.bio">https://openproblems.bio</a>
</div>
<div id="ref-pearson1901pca" class="csl-entry" role="doc-biblioentry">
Pearson, K. (1901). On lines and planes of closest fit to systems of points in space. <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em>, <em>2</em>(11), 559–572. <a href="https://doi.org/10.1080/14786440109462720">https://doi.org/10.1080/14786440109462720</a>
</div>
<div id="ref-pliner2019supervised" class="csl-entry" role="doc-biblioentry">
Pliner, H. A., Shendure, J., &amp; Trapnell, C. (2019). Supervised classification enables rapid annotation of cell atlases. <em>Nature Methods</em>, <em>16</em>(10), 983–986. <a href="https://doi.org/10.1038/s41592-019-0535-3">https://doi.org/10.1038/s41592-019-0535-3</a>
</div>
<div id="ref-polanski2020bbknn" class="csl-entry" role="doc-biblioentry">
Polański, K., Young, M. D., Miao, Z., Meyer, K. B., Teichmann, S. A., &amp; Park, J.-E. (2019). <span>BBKNN</span>: Fast batch alignment of single cell transcriptomes. <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btz625">https://doi.org/10.1093/bioinformatics/btz625</a>
</div>
<div id="ref-raredon2022computation" class="csl-entry" role="doc-biblioentry">
Raredon, M. S. B., Yang, J., Garritano, J., Wang, M., Kushnir, D., Schupp, J. C., et al. (2022). Computation and visualization of cell–cell signaling topologies in single-cell systems data using connectome. <em>Scientific Reports</em>, <em>12</em>(1). <a href="https://doi.org/10.1038/s41598-022-07959-x">https://doi.org/10.1038/s41598-022-07959-x</a>
</div>
<div id="ref-rodriques2019slide" class="csl-entry" role="doc-biblioentry">
Rodriques, S. G., Stickels, R. R., Goeva, A., Martin, C. A., Murray, E., Vanderburg, C. R., et al. (2019). Slide-seq: A scalable technology for measuring genome-wide expression at high spatial resolution. <em>Science</em>, <em>363</em>(6434), 1463–1467. <a href="https://doi.org/10.1126/science.aaw1219">https://doi.org/10.1126/science.aaw1219</a>
</div>
<div id="ref-sarkar2021separating" class="csl-entry" role="doc-biblioentry">
Sarkar, A., &amp; Stephens, M. (2021). Separating measurement and expression models clarifies confusion in single-cell <span>RNA</span> sequencing analysis. <em>Nature Genetics</em>, <em>53</em>(6), 770–777. <a href="https://doi.org/10.1038/s41588-021-00873-4">https://doi.org/10.1038/s41588-021-00873-4</a>
</div>
<div id="ref-schober2018correlation" class="csl-entry" role="doc-biblioentry">
Schober, P., Boer, C., &amp; Schwarte, L. A. (2018). Correlation coefficients. <em>Anesthesia &amp; Analgesia</em>, <em>126</em>(5), 1763–1768. <a href="https://doi.org/10.1213/ane.0000000000002864">https://doi.org/10.1213/ane.0000000000002864</a>
</div>
<div id="ref-stanley2020harmonic" class="csl-entry" role="doc-biblioentry">
Stanley, J. S., Gigante, S., Wolf, G., &amp; Krishnaswamy, S. (2020). Harmonic alignment. In <em>Proceedings of the 2020 <span>SIAM</span> international conference on data mining</em> (pp. 316–324). Society for Industrial; Applied Mathematics. <a href="https://doi.org/10.1137/1.9781611976236.36">https://doi.org/10.1137/1.9781611976236.36</a>
</div>
<div id="ref-stoeckius2017simultaneous" class="csl-entry" role="doc-biblioentry">
Stoeckius, M., Hafemeister, C., Stephenson, W., Houck-Loomis, B., Chattopadhyay, P. K., Swerdlow, H., et al. (2017). Simultaneous epitope and transcriptome measurement in single cells. <em>Nature Methods</em>, <em>14</em>(9), 865–868. <a href="https://doi.org/10.1038/nmeth.4380">https://doi.org/10.1038/nmeth.4380</a>
</div>
<div id="ref-stuart2019comprehensive" class="csl-entry" role="doc-biblioentry">
Stuart, T., Butler, A., Hoffman, P., Hafemeister, C., Papalexi, E., Mauck, W. M., et al. (2019). Comprehensive integration of single-cell data. <em>Cell</em>, <em>177</em>(7), 1888–1902.e21. <a href="https://doi.org/10.1016/j.cell.2019.05.031">https://doi.org/10.1016/j.cell.2019.05.031</a>
</div>
<div id="ref-szubert2019structurepreserving" class="csl-entry" role="doc-biblioentry">
Szubert, B., Cole, J. E., Monaco, C., &amp; Drozdov, I. (2019). Structure-preserving visualisation of high dimensional single-cell datasets. <em>Scientific Reports</em>, <em>9</em>(1). <a href="https://doi.org/10.1038/s41598-019-45301-0">https://doi.org/10.1038/s41598-019-45301-0</a>
</div>
<div id="ref-tabula2018single" class="csl-entry" role="doc-biblioentry">
Tabula Muris Consortium. (2018). Single-cell transcriptomics of 20 mouse organs creates a tabula muris. <em>Nature</em>, <em>562</em>(7727), 367–372. <a href="https://doi.org/10.1038/s41586-018-0590-4">https://doi.org/10.1038/s41586-018-0590-4</a>
</div>
<div id="ref-tabula2020single" class="csl-entry" role="doc-biblioentry">
Tabula Muris Consortium. (2020). A single-cell transcriptomic atlas characterizes ageing tissues in the mouse. <em>Nature</em>, <em>583</em>(7817), 590–595. <a href="https://doi.org/10.1038/s41586-020-2496-1">https://doi.org/10.1038/s41586-020-2496-1</a>
</div>
<div id="ref-tasic2016adult" class="csl-entry" role="doc-biblioentry">
Tasic, B., Menon, V., Nguyen, T. N., Kim, T. K., Jarsky, T., Yao, Z., et al. (2016). Adult mouse cortical cell taxonomy revealed by single cell transcriptomics. <em>Nature Neuroscience</em>, <em>19</em>(2), 335–346. <a href="https://doi.org/10.1038/nn.4216">https://doi.org/10.1038/nn.4216</a>
</div>
<div id="ref-tian2019benchmarking" class="csl-entry" role="doc-biblioentry">
Tian, L., Dong, X., Freytag, S., Cao, K.-A. L., Su, S., JalalAbadi, A., et al. (2019). Benchmarking single cell <span>RNA</span>-sequencing analysis pipelines using mixture control experiments. <em>Nature Methods</em>, <em>16</em>(6), 479–487. <a href="https://doi.org/10.1038/s41592-019-0425-8">https://doi.org/10.1038/s41592-019-0425-8</a>
</div>
<div id="ref-vandermaaten2008visualizing" class="csl-entry" role="doc-biblioentry">
van der Maaten, L., &amp; Hinton, G. (2008). Visualizing data using t-SNE. <em>Journal of Machine Learning Research</em>, <em>9</em>(86), 2579–2605. <a href="http://jmlr.org/papers/v9/vandermaaten08a.html">http://jmlr.org/papers/v9/vandermaaten08a.html</a>
</div>
<div id="ref-venna2001neighborhood" class="csl-entry" role="doc-biblioentry">
Venna, J., &amp; Kaski, S. (2001). Neighborhood preservation in nonlinear projection methods: An experimental study. In <em>Artificial neural networks <span>—</span> <span>ICANN</span> 2001</em> (pp. 485–491). Springer Berlin Heidelberg. <a href="https://doi.org/10.1007/3-540-44668-0_68">https://doi.org/10.1007/3-540-44668-0_68</a>
</div>
<div id="ref-venna2006local" class="csl-entry" role="doc-biblioentry">
Venna, J., &amp; Kaski, S. (2006). Local multidimensional scaling. <em>Neural Networks</em>, <em>19</em>(6-7), 889–899. <a href="https://doi.org/10.1016/j.neunet.2006.05.014">https://doi.org/10.1016/j.neunet.2006.05.014</a>
</div>
<div id="ref-wagner2018single" class="csl-entry" role="doc-biblioentry">
Wagner, D. E., Weinreb, C., Collins, Z. M., Briggs, J. A., Megason, S. G., &amp; Klein, A. M. (2018). Single-cell mapping of gene expression landscapes and lineage in the zebrafish embryo. <em>Science</em>, <em>360</em>(6392), 981–987. <a href="https://doi.org/10.1126/science.aar4362">https://doi.org/10.1126/science.aar4362</a>
</div>
<div id="ref-wagner2018knearest" class="csl-entry" role="doc-biblioentry">
Wagner, F., Yan, Y., &amp; Yanai, I. (2018). K-nearest neighbor smoothing for high-throughput single-cell RNA-seq data. <em>bioRxiv</em>. <a href="https://doi.org/10.1101/217737">https://doi.org/10.1101/217737</a>
</div>
<div id="ref-wang2013target" class="csl-entry" role="doc-biblioentry">
Wang, S., Sun, H., Ma, J., Zang, C., Wang, C., Wang, J., et al. (2013). Target analysis by integration of transcriptome and <span>ChIP</span>-seq data with <span>BETA</span>. <em>Nature Protocols</em>, <em>8</em>(12), 2502–2515. <a href="https://doi.org/10.1038/nprot.2013.150">https://doi.org/10.1038/nprot.2013.150</a>
</div>
<div id="ref-welch2019single" class="csl-entry" role="doc-biblioentry">
Welch, J. D., Kozareva, V., Ferreira, A., Vanderburg, C., Martin, C., &amp; Macosko, E. Z. (2019). Single-cell multi-omic integration compares and contrasts features of brain cell identity. <em>Cell</em>, <em>177</em>(7), 1873–1887.e17. <a href="https://doi.org/10.1016/j.cell.2019.05.006">https://doi.org/10.1016/j.cell.2019.05.006</a>
</div>
<div id="ref-wu2021single" class="csl-entry" role="doc-biblioentry">
Wu, S. Z., Al-Eryani, G., Roden, D. L., Junankar, S., Harvey, K., Andersson, A., et al. (2021). A single-cell and spatially resolved atlas of human breast cancers. <em>Nature Genetics</em>, <em>53</em>(9), 1334–1347. <a href="https://doi.org/10.1038/s41588-021-00911-1">https://doi.org/10.1038/s41588-021-00911-1</a>
</div>
<div id="ref-xiong2020neuralee" class="csl-entry" role="doc-biblioentry">
Xiong, J., Gong, F., Wan, L., &amp; Ma, L. (2020). <span>NeuralEE</span>: A <span>GPU</span>-accelerated elastic embedding dimensionality reduction method for visualizing large-scale <span class="nocase">scRNA</span>-seq data. <em>Frontiers in Genetics</em>, <em>11</em>. <a href="https://doi.org/10.3389/fgene.2020.00786">https://doi.org/10.3389/fgene.2020.00786</a>
</div>
<div id="ref-xiong2021online" class="csl-entry" role="doc-biblioentry">
Xiong, L., Tian, K., Li, Y., Ning, W., Gao, X., &amp; Zhang, Q. C. (2022). Online single-cell data integration through projecting heterogeneous datasets into a common cell-embedding space. <em>Nature Communications</em>, <em>13</em>(1). <a href="https://doi.org/10.1038/s41467-022-33758-z">https://doi.org/10.1038/s41467-022-33758-z</a>
</div>
<div id="ref-xu2021probabilistic" class="csl-entry" role="doc-biblioentry">
Xu, C., Lopez, R., Mehlman, E., Regier, J., Jordan, M. I., &amp; Yosef, N. (2021). Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. <em>Molecular Systems Biology</em>, <em>17</em>(1). <a href="https://doi.org/10.15252/msb.20209620">https://doi.org/10.15252/msb.20209620</a>
</div>
<div id="ref-zhang2021pydrmetrics" class="csl-entry" role="doc-biblioentry">
Zhang, Y., Shang, Q., &amp; Zhang, G. (2021). <span class="nocase">pyDRMetrics</span> - a python toolkit for dimensionality reduction quality assessment. <em>Heliyon</em>, <em>7</em>(2), e06199. <a href="https://doi.org/10.1016/j.heliyon.2021.e06199">https://doi.org/10.1016/j.heliyon.2021.e06199</a>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>